data/repositories.toml

```diff
@@ -764,4 +764,5 @@ repositories = [
   'github.com/ZupIT/ritchie-cli',
   'github.com/ZupIT/ritchie-formulas',
   'github.com/bmarvinb/react-trello-clone',
+  'github.com/datafuselabs/datafuse',
```

Comment:
Hey @zhang2014 Please add the repository in lexicographic order.

Reply:
Thanks your review. It's done.